packages/lesswrong/components/posts/PostsItem2.jsx

```diff
@@ -271,17 +339,29 @@ class PostsItem2 extends PureComponent {
               <PostsItemIcons post={post}/>
             </Hidden>}
 
-            <div className={classes.commentsIcon}>
-              <PostsItemComments 
-                post={post} 
-                onClick={() => this.toggleComments(false)} 
+            {!resumeReading && <div className={classes.commentsIcon}>
+              <PostsItemComments
+                post={post}
+                onClick={() => this.toggleComments(false)}
                 unreadComments={unreadComments}
               />
-            </div>
-
+            </div>}
+            
+            {resumeReading &&
+              <div className={classes.sequenceImage}>
+                <NoSSR>
+                  <Components.CloudinaryImage
+                    publicId={resumeReading.sequence?.gridImageId || resumeReading.collection?.gridImageId || "sequences/vnyzzznenju0hzdv6pqb.jpg"}
+                    height={48}
+                    width={146}
+                  />
+                </NoSSR>
+              </div>}
           </div>
+          
           {<div className={classes.actions}>
-            <PostsPageActions post={post} vertical menuClassName={classes.actionsMenu} />
+            {resumeReading && <CloseIcon onClick={() => this.props.dismissRecommendation()}/>}
```

Comment:
Destructure from props instead of referencing props directly.

Also, I would remove the unnecessary closure and just pass `dismissRecommendations` directly to the onClick property.

Reply:
It isn't necessary in this case, but as convention, I prefer we always make a closure when using an onFoo attribute, because it's way too easy to end up with `this` bound incorrectly otherwise..

Reply:
Hmm, ok. That seems fine.